src/PatchOrchestrationApplication/CoordinatorService/src/RepairManagerHelper.cs

```diff
@@ -274,6 +279,214 @@ internal async Task PrintRepairTasks(CancellationToken cancellationToken)
+                        propertyDict.Add(WUOperationStatusUpdate + "-" + node.NodeName, true);
+                    }
+                    foreach (var e in healthEventsToCheck)
+                    {
+                        if (!propertyDict.ContainsKey(e.HealthInformation.Property))
+                        {
+                            orphanProperties.Add(e.HealthInformation.Property);
+                        }
+                    }
+
+                    foreach (var property in orphanProperties)
+                    {
+                        ServiceEventSource.Current.VerboseMessage("Property {0}'s event is removed from CoordinatorService", property);
+
+                        // I think we would need to change the expiry time to ~0
+                        string description = "This health event will be expired in 1 minute as node corresponding to this event is deleted.";
+                        HealthManagerHelper.PostNodeHealthReport(fabricClient, this.context.ServiceName, property, description, HealthState.Ok, 1);
+                    }
+                }
+            }
+            catch(Exception ex)
+            {
+                ServiceEventSource.Current.ErrorMessage("ClearOrphanEvents failed with exception {0}", ex.ToString());
+            }
+        }
+
+        /// <summary>
+        /// Posts the cluster patching status by finding the nodes on which patching is going on.
+        /// </summary>
+        private async Task PostRMTaskNodeUpdate(CancellationToken cancellationToken)
+        {
+            NodeList nodeList = await this.fabricClient.QueryManager.GetNodeListAsync(null, null, this.DefaultTimeoutForOperation, cancellationToken);
+            HashSet<string> processingNodes = new HashSet<string>();
+            HashSet<string> pendingNodes = new HashSet<string>();
+            IList<RepairTask> claimedTaskList = await this.GetClaimedRepairTasks(nodeList, cancellationToken);
+            foreach (var task in claimedTaskList)
+            {
+                pendingNodes.Add(task.Target.ToString());
+            }
+            cancellationToken.ThrowIfCancellationRequested();
+            RepairTaskList processingTaskListFinal = await this.GetRepairTasksUnderProcessing(cancellationToken);
+            foreach (var task in processingTaskListFinal)
+            {
+                processingNodes.Add(task.Target.ToString());
+            }
+
+            string pendingNodesString = string.Join(",", pendingNodes);
+            string processingNodesString = string.Join(",", processingNodes);
+            
+            if(String.IsNullOrEmpty(pendingNodesString))
+            {
+                pendingNodesString = "None";
+            }
+
+            if (String.IsNullOrEmpty(processingNodesString))
+            {
+                processingNodesString = "None";
+            }
+
+            string description = string.Format("ProcessingNodes :{0}, PendingNodes: {1}", processingNodesString, pendingNodesString);
```

Comment:
>, PendingNodes: [](start = 68, length = 15)

Can we show the two lists in two different lines?

Reply:
yes